package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "datamegh",
+  "version": "1.0.0-alpha.1",
+  "description": "Datamegh - Data Engineering for the cloud.",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "npx pyright"
```

Just this should do the job too - `npm` can already do this since you've installed the package locally here. 
```suggestion
    "test": "pyright"
```

---

Only if it was globally installed.

---

No, actually if you run it in this fashion with `npm` like `npm run typecheck` or `yarn typecheck` it can run both local and global scripts.